build.sbt

```diff
@@ -5,9 +5,7 @@ name := "identity-play-auth"
 
 organization := "com.gu.identity"
 
-scalaVersion := "2.11.11"
-
-crossScalaVersions := Seq(scalaVersion.value, "2.12.4")
+scalaVersion := "2.12.4"
```

Comment:
2.12.6 looks like the latest version - not sure if it's better to use that? https://www.scala-lang.org/news/2.12.6